app/models/user.rb

```diff
@@ -3,8 +3,51 @@ class User < ActiveRecord::Base
   # validates password and password_confirmation; no validates_presence_of necessary!!!
   has_secure_password validations: true
   has_many :queue_items, -> { order('order_value') }
+  has_many :reviews
 
   validates_presence_of :email, :full_name
   validates_uniqueness_of :email
 
+  def update_queue_items(queue_items_hash)
```

Comment:
Pushing this down to the model layer is actually considered as bad practice. I am referring to [this assignment](https://www.gotealeaf.com/lessons/00bcc3b5/assignments/1569) especially solution video from 0:17 to 1:07.